packages/ui/src/design-system/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -37,13 +45,15 @@ export const Breadcrumbs = ({buttons, size, onClick}: BreadcrumbsProps) => {
             className="flex items-center shrink-0 gap-x-1"
           >
             <Button
```

Comment:
Pourquoi ne pas avoir garder l'ancienne structure `link.path ? <Link /> : <div/>` ?

Ca permet de ne pas avoir à styliser le `Button` pour fake un aspect `disabled` ou de `disabled` un bouton tout court.

D'ailleurs ça ne devrait pas être des `Button` mais des `Link`, car un bouton avec href, cela créé un `<a>` qui lorsque l'user click dessus, recharge toute la page au lieu de se servir du `router` de l'app. J'ai noté ça sur les compos avec des boutons et `href` de setup dessus, faut absolument qu'on trouve une solution.

Sur le nommage je mettrais `links` ou `items` pour etre plus générique à la place de `buttons` .

Reply:
Je ne peux pas utiliser le composant `<Link/>`, car selon le projet il sera importé soit de `react-router-dom`, soit de `next/link`. D'ailleurs la balise `<Link/>` crée aussi une balise `<a/>`. Et je ne pense pas qu'il faille s'appuyer sur `react-router-dom` dans les composants DS alors qu'on prépare la migration vers next.

Pour les boutons avec `href` on pourrait peut être régler le problème avec `router.push` ou `history.push` depuis le `onClick` en attendant de pouvoir l'intégrer directement au composant après la migration de l'app (je n'étais pas encore tombée sur ce problème).

Sinon j'ai utilisé le composant `Button` partout pour réutiliser ce qui est déjà fait en terme de fonctionnalité avec `href` et `onClick`, mais aussi pour utiliser les mises en formes déjà faites (padding, taille de texte et espacements en fonction de la taille de bouton choisie). Si j'utilise une `<div/>` je dois refaire toute la mise en forme, c'est plus simple de mettre un bouton en disabled.

Ok pour le nommage, je remplace par `items` du coup.

Reply:
Je sais c'est relou pour les balises `<Link />` :/

Oui je pense que c'est une bonne option le `onClick` avec le fonction de framework donnée par l'app qui l'utilise.
Tu fais un test avec ce compo?

okok pour la <div /> 👍

Reply:
J'ai fait la mise à jour, ça a l'air de bien marcher 🙂